### ISSUE TITLE
uwsgi/ImportError: cannot import name CSRFProtect

### ISSUE THREAD

**Issue by jasonblanks** [google/timesketch#301]
Hi,  this might be out of scope, but thought id drop this here.  Trying to get timesketch to play well with uwsgi, all new to me so i might be doing something stupid, but has anyone tried doing the same and get it working noticed some dev comments with uwsgi and nginx?    

uwsgi --http :5000 --wsgi-file timesketch/wsgi.py

Traceback (most recent call last):
  File "timesketch/wsgi.py", line 35, in <module>
    from timesketch import create_app
  File "./timesketch/__init__.py", line 24, in <module>
    from flask_wtf import CSRFProtect
ImportError: cannot import name CSRFProtect
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 25031, cores: 1)

-----
the wsgi file gave a gunicorn example, so I gave that a shot too:

[2017-03-20 12:54:06 +0000] [25061] [INFO] Starting gunicorn 19.7.0
[2017-03-20 12:54:06 +0000] [25061] [INFO] Listening at: http://127.0.0.1:5000 (25061)
[2017-03-20 12:54:06 +0000] [25061] [INFO] Using worker: sync
[2017-03-20 12:54:06 +0000] [25066] [INFO] Booting worker with pid: 25066
[2017-03-20 12:54:06 +0000] [25066] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/arbiter.py", line 578, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 126, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 135, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 65, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/util.py", line 376, in import_app
    __import__(module)
  File "/data/dev/timesketch/timesketch/__init__.py", line 24, in <module>
    from flask_wtf import CSRFProtect
ImportError: cannot import name CSRFProtect
[2017-03-20 12:54:06 +0000] [25066] [INFO] Worker exiting (pid: 25066)
[2017-03-20 12:54:06 +0000] [25061] [INFO] Shutting down: Master
[2017-03-20 12:54:06 +0000] [25061] [INFO] Reason: Worker failed to boot.

------------------------------------------

Name: Flask-WTF
Version: 0.13.1
Summary: Simple integration of Flask and WTForms.
Home-page: https://github.com/lepture/flask-wtf
Author: Hsiaoming Yang
Author-email: me@lepture.com
License: BSD
Location: /usr/local/lib/python2.7/dist-packages
Requires: Flask, Werkzeug, WTForms

**Comment by jasonblanks** [comment]
For whatever reason without any system or lib changes this wanted to work today.  Started to try to trouble shoot this again this morning and started without issues w. gunicorn and uwsgi.  -\-(-_-)_/-.

**Comment by thewilliamzhang** [comment]
I had the same issue. The problem comes from dependency version incompatibility. Do `pip install Flask-WTF==0.14` may help you resolve this.
